src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,25 @@
+package frc.robot.subsystems;
+
+import com.flash3388.flashlib.scheduling.Subsystem;
+import com.revrobotics.CANSparkMax;
+
+public class Intake extends Subsystem {
+    private CANSparkMax motor;
+    private final double SPEED = -0.2;
```

Comment:
```suggestion
    private static final double SPEED = -0.2;
```

For a true constant `static final` are both required as modifiers.